lib/active_shipping/carriers/fedex.rb

```diff
@@ -153,10 +161,129 @@ def find_tracking_info(tracking_number, options = {})
       parse_tracking_response(xml, options)
     end
 
+
+    # Get Shipping labels
+    # Caveats:
+    #  - Only supports singlular packages
```

Comment:
I'm not intimately familiar with the UPS or FedEx APIs anymore, but I seem to recall that a single UPS shipment supports the notion of a single tracking number for a set of packages.  I'm pretty sure `map` would work in practice and is arguably more elegant, but it could leading to billing problems if discounts have been negotiated for multi-package shipments.